src/upload/upload.ts

```diff
@@ -0,0 +1,43 @@
+import { MongoClient, Collection } from "mongodb";
+
+type documents = Array<any>;
```

Comment:
Can we shift to using a JSON Interface?
```suggestion
interface MakeshiftJSONInterface {
  [key: string]:
    | string
    | number
    | boolean
    | null
    | MakeshiftJSONInterface
    | Array<MakeshiftJSONInterface>;
}
type documents = Array<MakeshiftJSONInterface>;
```

Reply:
Done. Please take a look.